### PR TITLE
docs(readme): add instructions for adding validator information

### PR DESCRIPTION
**Pull request by onmax**

### CODE REVIEW

**Review by Albermonte — APPROVED** [review]

**Review by Albermonte — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,6 +12,19 @@ Details of validators in the Nimiq Blockchain and their scores, calculated using
...
 > If you're a validator and would like to add your data to the API, please open a pull request (PR) with your JSON file, following the structure of `./public/validators/.example` in the `./public/validators` directory.
 
+## Add your validator information
+
+If you run your own validator and want it to be recognised, you can add a name, logo and other publicly available fields. For example, the Nimiq Wallet will use this information to display your validator to users.
+
+In order to do so, follow these steps:
+
+1. [Fork this repository](https://github.com/nimiq/validators-api/fork).
+2. Create a new file in the [validators](./public/validators/) folder. You can use `.example.json` as a reference.
```
Comment (Eligioo) [inline README.md]: I had a look at the example.json and not all fields were quite clear to me:
- icon: Should this icon be added to the repo too? Or via a publicly reachable domain?
- fee: How is this field interpreted? Can it be int or decimals? Does a value of 1 mean 1% fee?
- isPool: Why is the default `null`? I would expect a boolean here.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,19 @@ Details of validators in the Nimiq Blockchain and their scores, calculated using
...
+3. Submit a PR to this repo. Someone in the team will review it.
+
+> [!INFO]
+> By submitting this information, various Nimiq applications, such as the Nimiq Wallet, will show users the fields you have submitted and the Validator Score associated with your validator in the future. You can read more about the Validator Score in the [Developer's Center PR](https://github.com/nimiq/developer-center/pull/21).
```
Comment (Eligioo) [inline README.md]: Linking to a PR isn't really helpful? Can't we link to the developer center directly?

Comment (onmax) [reply README.md]: It is not merged yet. People reading these instructions are quite familiar already with GitHub so I thought it wouldn't be a problem :/

File: README.md (hunk 1 above)
Comment (onmax) [reply README.md]: Ok, I can change this file to:

```
{
     "icon": "Optional. A base64 encoded SVG icon. It should start with `data:image/svg+xml;base64...`. If you are not sure how to get this value, you can leave this field empty and add the svg in the PR. One of the maintainers will convert it for you :)",
     "fee": 0.1,
     "isPool": false
}
```

I want to keep this field JSON-compatible, so I will move the comments also to this readme

**Comment by onmax** [comment]
@Eligioo @sisou 


What do you think about storing the boolean `isPool` instead of the `payoutType: 'direct' | 'restake'` as we do in the wallet. Does this change make sense, or is it better to keep `payoutType`? Is this future-proof?


Wallet code: https://github.com/nimiq/wallet/blob/a275f048651dd3714a638a55c081d21f1ba2169d/src/stores/Prestaking.ts#L30

File: README.md (hunk 1 above)
Comment (Eligioo) [reply README.md]: Alright 👍

File: README.md (hunk 2 above)
Comment (Eligioo) [reply README.md]: You are looking at raw markup and git changes here. Not exactly pretty but just my 2 cents.

**Comment by Eligioo** [comment]
> @Eligioo @sisou
> 
> What do you think about storing the boolean `isPool` instead of the `payoutType: 'direct' | 'restake'` as we do in the wallet. Does this change make sense, or is it better to keep `payoutType`? Is this future-proof?
> 
> Wallet code: https://github.com/nimiq/wallet/blob/a275f048651dd3714a638a55c081d21f1ba2169d/src/stores/Prestaking.ts#L30

In my opinion enums are better suited for this.

**Comment by sisou** [comment]
So like an enum that has `direct`, `restake` and `none` for non-pools?

File: README.md (hunk 2 above)
Comment (sisou) [reply README.md]: The preview link in the PR description doesn't work.

**Comment by onmax** [comment]
> So like an enum that has direct, restake and none for non-pools?


Yes, sounds good. I need to update the SQL and logic and update this same pr

File: README.md [hunk 3]
```diff
@@ -12,6 +12,40 @@ Details of validators in the Nimiq Blockchain and their scores, calculated using
...
+
+If you run your own validator and want it to be recognised, you can add a name, logo and other publicly available fields. For example, the Nimiq Wallet will use this information to display your validator to users.
+
+In order to do so, follow these steps:
+
+1. [Fork this repository](https://github.com/nimiq/validators-api/fork).
+2. Create a new file in the [validators](./public/validators/) folder. You can use `.example.json` as a reference.
+
+<details>
+  DCFER32W1Q<tab>	YTRFEDWSQAAAAAAAAAAAAAAAAAAAAAA
```
Comment (Eligioo) [inline README.md]: A cat walked over your keyboard? 🐱

File: README.md [hunk 4]
```diff
@@ -12,6 +12,40 @@ Details of validators in the Nimiq Blockchain and their scores, calculated using
...
+  Validator JSON fields
+  </summary>
+
+When you add your validator information, you'll need to include certain keys in your JSON file. Below is an explanation of each key and its possible values:
+
+- `address` (required): The address of the validator in the format `NQXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX`.
+- `fee` (Required): A number between -1 and 1 representing the fee percentage charged by your validator. For example, 0.05 represents a 5% fee.
```
Comment (Eligioo) [inline README.md]: A number between 0 and 1? I suppose -1 when there is no fee?

File: README.md [hunk 5]
```diff
@@ -12,6 +12,40 @@ Details of validators in the Nimiq Blockchain and their scores, calculated using
...
+- `payoutType` (required): The method used to pay out validator rewards. Possible values are
+  - `restake`: Rewards are automatically restaked.
+  - `direct`: Rewards are paid directly to the validator.
```
Comment (Eligioo) [inline README.md]:
```suggestion
  - `direct`: Rewards are paid directly to the staker's wallet and not automatically restaked.
```

File: README.md [hunk 6]
```diff
@@ -12,6 +12,40 @@ Details of validators in the Nimiq Blockchain and their scores, calculated using
...
+  - `none`: No rewards will be paid out.
+- `name` (optional): The name of the validator. If not specified, the address of the validator is used by default.
+- `description` (optional): A short description of your validator.
+- `website` (optional): The URL of your validator's website.
+- `icon` (optional): A base63 encoded SVG icon representing your validator. It is preferable if it has Nimiq's hexagon shape.
```
Comment (Eligioo) [inline README.md]:
```suggestion
- `icon` (optional): A base64 encoded SVG icon representing your validator. The logo being in Nimiq's hexagon shape is preferred.
```

File: public/validators/.example.json [hunk 7]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "Optional. The name of the validator, by default it is the address",
   "description": "Optional. A description of the validator",
-  "website": "Optional. The website of the validator",
+  "website": "Optional. The website of the validator. By default identicon is used. Preferable if it has an Nimiq's hexagon-shape",
```
Comment (Eligioo) [inline public/validators/.example.json]: The last bit of this text belongs to the `icon` property of the JSON?

File: README.md (hunk 6 above)
Comment (sisou) [reply README.md]: Representing your **pool**.

**Pull request merged**